### ISSUE TITLE
NPE in TransferService after update to 2.2.14

### ISSUE THREAD

**Issue by niqo01** [aws/aws-sdk-android#118]
```
Fatal Exception: java.lang.NullPointerException
       at com.amazonaws.mobileconnectors.s3.transferutility.TransferService.execCommand(TransferService.java:287)
       at com.amazonaws.mobileconnectors.s3.transferutility.TransferService$UpdateHandler.handleMessage(TransferService.java:224)
       at android.os.Handler.dispatchMessage(Handler.java:110)
       at android.os.Looper.loop(Looper.java:193)
       at android.os.HandlerThread.run(HandlerThread.java:61)
```
See all details here:
http://crashes.to/s/84e42fce62c

**Comment by fosterzhang** [comment]
We've identified the bug. Please hold off the update and wait for next release.

**Comment by fosterzhang** [comment]
This is fixed in v2.2.15. See [release notes](http://aws.amazon.com/releasenotes/Android/9052762089998010).
